src/riak_kv_qry_compiler.erl

```diff
@@ -319,13 +374,13 @@ merge_validation({error, E1}, {error, E2}) -> {error, E1 ++ E2}.
 
 %%
 compile_select_col_stateless2('+', A, B) ->
-    fun(Row) -> A(Row) + B(Row) end;
+    fun(Row, State) -> A(Row, State) + B(Row, State) end;
```

Comment:
That variable is more like a `ColValue` than a state, or do you think there is a better name? IMHO the function is correctly named as it purposely does not handle aggregate functions.